selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/types/RenameTypeTest.java

```diff
@@ -70,17 +69,17 @@ public void setup() throws Exception {
     testProjectServiceClient.importProject(
         workspace.getId(),
         Paths.get(resource.toURI()),
-        nameOfProject,
+        NAME_OF_PROJECT,
         ProjectTemplates.MAVEN_SIMPLE);
     ide.open(workspace);
-    projectExplorer.waitVisibleItem(nameOfProject);
+    projectExplorer.waitVisibleItem(NAME_OF_PROJECT);
     consoles.closeProcessesArea();
     projectExplorer.quickExpandWithJavaScript();
     loader.waitOnClosed();
   }
 
   @BeforeMethod
-  public void setCurrentFieldForTest(Method method) throws IOException {
+  public void setCurrentFieldForTest(Method method) {
     try {
       setFieldsForTest(method.getName());
     } catch (Exception e) {
```

Comment:
It's strange - we are hiding exception here. Is it really correct behavior?

Reply:
As i guess this have been done for avoiding with skipping in tthe @Before method annotation. But i can not remember the exact cause